test_requirements.txt

```diff
@@ -1,5 +1,5 @@
 pytest
 decorator
 pylint
-black==19.10b0
+black
```

Comment:
I'm being a bit pedantic now, but that is the best I can do when we are to apply a uncompromising code formatter 🤷 I don't think this, nor the removal of the formatting test should be touched in this commit. I would like it to purely be the result of applying black to the source code.

Maybe we can do something like this with the commits(?):
- Bump black to 19.10b0 (and update currently blacked lines)
- Apply black on all files (<- point is I want this to be a PR with no human edits)
- Move black test out of test_formatting and into CI

Reply:
Done :D